app/templates/sectionsummary.html

```diff
@@ -0,0 +1,38 @@
+{% import 'macros/helpers.html' as helpers %}
+
+{% extends theme('section_confirmation.html') -%}
+
+{% block page_title -%}{{_("Section-Summary")}} - {{survey_title}}{% endblock -%}
+
+{% block sidebar %}
+  {% if navigation %}
+    {% include theme('partials/navigation.html') %}
+  {% endif %}
+{% endblock %}
+
+{% block form_content %}
+{% endblock %}
+
+{% block main -%}
+
+<div>
+    {% for group in content.summary.groups %}
+        <h1 class="summary__title saturn" id="{{group.id}}">{{group.title}}</h1>
```

Comment:
If `group.title` is an empty string then this will render an empty `<h2>`. There's a check in `summary.html` to prevent this - could we add the check in here as well please?

Reply:
@MIslam92 we now arn't checking that there is a title